specs/binding-http.spec/src/main/resources/META-INF/zilla/http.idl

```diff
@@ -49,4 +49,31 @@ scope http
             HttpHeader[] headers;
         }
     }
+
+    scope event
+    {
+        enum HttpEventType (uint8)
+        {
+            AUTHORIZATION (1),
+            ACCESS_DENIED (2)
```

Comment:
Is there any difference between `AUTHORIZED` + `FAILURE` and `ACCESS_DENIED`?

If not, let's go with `ACCESS_ALLOWED` and `ACCESS_DENIED` event types, both using `HttpAccessEvent` struct to indicate `identity`.

Do we also need `method`,`scheme`, `authority` and `path` to be able to present a useful access log?

Reply:
`ACCESS_DENIED` is being called from here:
https://github.com/aklivity/zilla/blob/a8e6e8c96ca1e2f6a87b5f6bfe3175c1f5820b33/runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/stream/HttpServerFactory.java#L1033

This one was renamed more than once already, and I'm still not sure that this is the best name for it. However, I believe that this event related to `isCorsRequestAllowed` is a different type of event than an authorization failure (which is a wrong username, password etc.) I mean, if we want to log this at all. Pls LMK.

I'd keep the name `AUTHORIZATION` for authorizaton success/failure events for the sake of consistency, because we have very similar authorization events in `http`, `mqtt` and `kafka` and it'd make sense to use the same name everywhere.

Reply:
Sure, just let's make sure to pick the same name in http, mqtt and kafka.

Reply:
> Yep understood, I don't think we currently need to log CORS rejection.

OK, removed.